test/DynamoCoreTests/Performance/PerformanceTests.cs

```diff
@@ -67,6 +140,13 @@ public void PerformanceTest(string filePath)
 
         private void DoWorkspaceOpenAndCompare(string filePath)
         {
+
+            WarmUp();
+
+            var hardCodedExecutionTime = RunHardcodedTest(filePath);
```

Comment:
@sm6srw perhaps I am very confused... does this test get run everytime a perf test file is run?